### ISSUE TITLE
Pulses Compressing Columns 

### ISSUE THREAD

**Issue by petesmuller** [metabase/metabase#8446]
Since our most recent update all of our pulses are essentially concatenating the first two columns such that they are unreadable. I'm on Metabase v0.30.1 and receiving the Pulses in gmail on both web and mobile. Example attached. 

![0d4a9acc-872f-4c12-b378-1297dfa44309](https://user-images.githubusercontent.com/38897467/45034020-4653ac00-b024-11e8-8083-db759b36d3c5.png)

-------

**Comment by senior** [comment]
Might be related to https://github.com/metabase/metabase/issues/8170

**Comment by ryanfoxsquire** [comment]
I just made an overdue upgrade from before .3 to the latest, and I
 have a similar but different issue -- I am seeing the 2nd and 3rd column squished together so that they are unreadable in a pulse. 

before(good): 


<img width="325" alt="screen shot 2018-09-07 at 5 44 33 pm" src="https://user-images.githubusercontent.com/6432906/45248474-f1cf5b80-b2c5-11e8-8236-4ff18f04668c.png">


now (bad):



<img width="304" alt="screen shot 2018-09-07 at 5 44 43 pm" src="https://user-images.githubusercontent.com/6432906/45248478-fb58c380-b2c5-11e8-97d4-2e59803be697.png">


This is really a bummer. I'm trying to hack around by adding in whitespace and things in the SQL but metabase automatically removes all whitespace, so there is no good workaround i can find.

**Comment by ryanfoxsquire** [comment]
I posted a semi-work around here: https://github.com/metabase/metabase/issues/8170#issuecomment-424455540

**Comment by mazameli** [comment]
So I'm pretty sure I figured out what's going on here, but my remedial Clojure knowledge is insufficient to figure out how to fix it. Maybe @paulrosenzweig or @camsaul can give me a quick assist?

What's going on is that only cells that have a background color (because of conditional formatting) are correctly getting assigned styling (`row-style-for-type`). Cells without any conditional formatting are not getting assigned any styling. Here's an inspector screengrab depicting this:

![image](https://user-images.githubusercontent.com/2223916/59879332-4fb5c700-935f-11e9-823e-fe961cf9182a.png)

I think the offending logic is here, starting at line 372 of `/src/metabase/pulse/render.clj`:

```
(map-indexed (fn [col-idx cell]
                                     (let [bg-color (color/get-background-color color-selector cell (get column-names col-idx) row-idx)]
                                       [:td {:style (style (row-style-for-type cell)
                                                           (merge {:background-color bg-color}
                                                                  (when (and bar-width (= col-idx 1))
                                                                    {:font-weight 700})))}
                                        (h cell)]))
                                   row)
```

**Comment by camsaul** [comment]
@mazameli can you do me a huge favor and see if you can repro this on master? I might have accidentally fixed this.

**Comment by mazameli** [comment]
@camsual Sure thing!

**Comment by mazameli** [comment]
@camsaul I'm not seeing this fixed on master. I pulled latest, restarted the server, rebuilt, then created a new pulse. You can see in the first table that the margins between columns aren't there, and the font size and type are wrong. In the second table, columns that have conditional formatting do have the right font size and type, and have correct spacing between columns:

![image](https://user-images.githubusercontent.com/2223916/60287445-c22a2800-98c6-11e9-9550-013d7b09a327.png)

**Comment by camsaul** [comment]
ok, thanks. I must have been trying to test it wrong

**Comment by walterl** [comment]
Fixed in #10650
